### ISSUE TITLE
route: support file extensions

### ISSUE THREAD

**Issue by francescoagati** [ufront/ufront-mvc#31]
actually the router don't support file extension. ins't possible make contet type negotiation using file extension

**Comment by jasononeil** [comment]
I agree it could be helpful to have:

    @:route("/manual/$page.html")
    @:route("/manual/$image.png")

as two separate routes.  Currently you'd need to do `@:route("/manual/$file")` and handle the different extensions in the same method.

If we are going to do this it's also worth asking if we will support general substrings as well:

     @:route("/manual/first-$part-last")
     @:route("/manual/start-$part-end")

This would be slightly more complex to implement, especially if we allow multiple captures:

     @:route("/manual/start-$part1-$part2-end")

Do you think that is needed, or mostly just supporting extensions would be helpful?

**Comment by francescoagati** [comment]
yes but if have see that  @:route("/manual/$file")  isn't trapped and return a 404. This on php.
